src/icons/Thread.tsx

```diff
@@ -0,0 +1,26 @@
+export default function () {
```

Comment:
Also, this should be called `Reply`, because clicking on it opens the text field, that will reply to the user